### PR TITLE
Add section on working with local memcached

### PR DESCRIPTION
**Pull request by bbaker6225**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -36,7 +36,24 @@ Using elasticsearch locally
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 To use elasticsearch locally, and to update your index after adding new data that you want elasticsearch to access
 run:
-$ ./manage.py update_index --disable-change-limit
+.. code-block:: shell
+
+    $ ./manage.py update_index --disable-change-limit
```
Comment (bbaker6225) [inline README.rst:42]: Just some better formatting for this line

File: README.rst [hunk 2]
```diff
@@ -36,7 +36,26 @@ Using elasticsearch locally
...
+
+.. code-block:: shell
+
+    $ ./manage.py update_index --disable-change-limit
+
+
+Working with memcached locally
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Some endpoints, such as /api/v1/courses, have their responses cached in memcached through the
+mechanisms such as the CompressedCacheResponseMixin. This caching may make it difficult to see code changes
```
Comment (mikix) [inline README.rst]: `the mechanisms such as the CompressedCacheResponseMixin` might read better without either of the `the`s.

File: README.rst [hunk 3]
```diff
@@ -36,7 +36,26 @@ Using elasticsearch locally
...
+reflected in various endpoints without first clearing the cache or updating the cache keys. To update the cache
+keys, make a change to any course_metadata model in the admin dashboard and save it. To flush your local memcached,
```
Comment (mikix) [inline README.rst]: Minor note: you don't need to make any changes. You just need to click `Save`

**Review by mikix — APPROVED** [review]
Nice

**Pull request merged**